CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 Changes
 =======
+
+# 1.0.0 / 01-07-2020
+### Details
+https://github.com/jenkinsci/datadog-plugin/compare/datadog-0.7.1...datadog-1.0.0
+
+### Changes
+[BACKWARDS INCOMPATIBILITY NTOES]
+* Instead of sending `null` as default value for some tags we now send `unknown`
+* Event messages got changed.
+* Node tag is set by default
```

Comment:
@gzussa Anything worth adding here? (Can you help clarify what about event messages is changed)

Reply:
Title and content of event messages got changed with more informations. Event search queries may not work anymore.